XIVComboExpanded/CustomComboPreset.cs

```diff
@@ -471,6 +471,9 @@ public enum CustomComboPreset
         // [CustomComboInfo("Kardia Into Soteria", "Kardia turns into Soteria when active and Soteria is off-cooldown.", SGE.JobID)]
         // SageKardiaFeature = 4001,
 
+        [CustomComboInfo("Druochole Into Taurochole Feature", "Taurochole changes into Druochole while Taurochole is on cooldown")]
```

Comment:
```suggestion
        [CustomComboInfo("Taurochole into Druochole Feature", "Taurochole changes into Druochole while Taurochole is on cooldown")]
```

Recommend reversing the combo label, just for clarity, since Taurochole now turns into Druochole in your updated version of the PR.